src/store/flow/block-types/SmartDevices_LocationResponseBlockStore.ts

```diff
@@ -34,12 +35,22 @@ const actions: ActionTree<IEmptyState, IRootState> = {
       prompt: blankMessageResource.uuid,
       accuracy_threshold_meters: 5.0,
       accuracy_timeout_seconds: 120,
+      ...await dispatch('initiateExtraVendorConfig'),
     }
+
+    props.exits = [
+      await dispatch('flow/block_createBlockDefaultExitWith', {
+        props: ({
+          uuid: await (new IdGeneratorUuidV4()).generate(),
+          name: 'Invalid',
```

Comment:
Took the opportunity to update LocationResponse branch too, to have `Valid/Invalid` exits instead of `1/Default`